src/main/java/po/WriteMessagePage.java

```diff
@@ -0,0 +1,33 @@
+package po;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class WriteMessagePage extends AbstractPage {
+
+    private static final By FIELD_WHOM = By.xpath("//div[@name='to']");
```

Comment:
@FindBy(xpath="//div[@name='to']")
private WebElement fieldWhom;

PageFactory.int(WriteMessagePage.class)